scripts/lib/api/generateApiComponents.ts

```diff
@@ -52,11 +275,7 @@ export async function createOpeningTag(
   const signature = await htmlSignatureToMd(props.rawSignature!);
   const extraSignatures: string[] = [];
   for (const sig of props.extraRawSignatures ?? []) {
-    extraSignatures.push(
-      `${APOSTROPHE_HEX_CODE}${await htmlSignatureToMd(
-        sig!,
-      )}${APOSTROPHE_HEX_CODE}`,
-    );
+    extraSignatures.push(`"${await htmlSignatureToMd(sig!)}"`);
```

Comment:
```suggestion
    extraSignatures.push(await htmlSignatureToMd(sig!));
```

Reply:
Notice that we have quotes surrounding the extra signature. We can't remove them because we need the signatures as a string inside the `extraSignatures` array. The result is something like `<Function ... extraSignatures = {["sig 1", "sig 2"]}>` where each signature has the quotes